src/Container/Docker/Credentials.hs

```diff
@@ -39,12 +45,24 @@ data DockerConfig = DockerConfig
 instance FromJSON DockerConfig where
   parseJSON = withObject "DockerConfig" $ \o ->
     DockerConfig
-      <$> o .: "credsStore"
+      <$> o .:? "credsStore"
+      <*> (o .:? "auths" .!= mempty)
       <*> o .:? "credHelpers" .!= mempty
```

Comment:
[nit]: Looks like unnecessary parens, since they're not needed below
```suggestion
      <*> o .:? "auths" .!= mempty
      <*> o .:? "credHelpers" .!= mempty
```